docs/getting-started/documentation/pull-data-transfer.md

```diff
@@ -72,17 +72,19 @@ To start a pull-http-transfer using the management-API of the EDC one can send t
 `POST` to `{{connector-base-url}}/control/data/transferprocess`
 ```json
 {
-  "protocol": "ids-multipart",
+  "@context": {
+    "edc": "https://w3id.org/edc/v0.0.1/ns/"
```

Comment:
this context was added, but no properties in this request start with `edc:`. This tells me that it might not work. You should really try out the request.